static/style.css

```diff
@@ -6,8 +6,8 @@
 }
 
 @media (min-width: 768px) {
-    .md\:max-w-\[1192px\] {
-        max-width: 1192px;
+    .md\:max-w-\[1210px\] {
```

Comment:
Can we stick with a `1366px` max width instead? It's what we're using for the blog, and is more of a [standardised screen size](https://en.wikipedia.org/wiki/Graphics_display_resolution#WXGA).